roles/competencies/software_engineering.md

```diff
@@ -0,0 +1,335 @@
+# SFIA 1 Role Guidance: Academy Software Engineer
+
+[SFIA Level 1: Follow](https://sfia-online.org/en/sfia-7/responsibilities/level-1)
```

Comment:
Perhaps the links should go to sfia-8 as that's what it currently is in the rest of the handbook now

Reply:
Yes, agreed.